test/potluck_test.rb

```diff
@@ -0,0 +1,99 @@
+require 'minitest/autorun'
+require 'minitest/emoji'
+require './lib/dish'
+require './lib/potluck'
+
+class DishTest < Minitest::Test
+
+  def test_it_exists
+    potluck = Potluck.new("7-13-18")
+  end
+
+  def test_it_holds_a_date
+    potluck = Potluck.new("7-13-18")
+    assert_equal "7-13-18", potluck.date
+  end
+
+  def test_it_starts_without_dishes
+    potluck = Potluck.new("7-13-18")
+    assert_equal [], potluck.dishes
+  end
+
+  def test_it_adds_dishes
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    potluck.add_dish(couscous_salad)
+    assert_instance_of Dish, potluck.dishes[0]
+  end
+
+  def test_it_holds_many_dishes
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+    potluck.add_dish(couscous_salad)
+    potluck.add_dish(cocktail_meatballs)
+    assert_instance_of Dish, potluck.dishes[1]
+  end
+
+  def test_it_counts_dishes
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+    potluck.add_dish(couscous_salad)
+    potluck.add_dish(cocktail_meatballs)
+    assert_equal 2, potluck.dishes.length
+  end
+
+  def test_it_can_be_accessed_by_category
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    summer_pizza = Dish.new("Summer Pizza", :appetizer)
+    roast_pork = Dish.new("Roast Pork", :entre)
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+    candy_salad = Dish.new("Candy Salad", :dessert)
+    potluck.add_dish(couscous_salad)
+    potluck.add_dish(summer_pizza)
+    potluck.add_dish(roast_pork)
+    potluck.add_dish(cocktail_meatballs)
+    potluck.add_dish(candy_salad)
+    expected = potluck.get_all_from_category(:appetizer).first.name
```

Comment:
Would probably call this `actual`, since it's the value that actually gets returned from the method.